### PR TITLE
[Task-4] : Containerization

### PR DESCRIPTION
**Pull request by zarutski**

### CODE REVIEW

File: resource/src/main/resources/application.properties [hunk 1]
```diff
@@ -1,8 +1,19 @@
-aws.s3.endpoint-url=http://localhost:4566
```
Comment (uladzimir-shelhunou-v2) [inline resource/src/main/resources/application.properties:1]: Use YAML.

File: resource/pom.xml [hunk 2]
```diff
@@ -75,6 +75,11 @@
             <version>3.1.3</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.postgresql</groupId>
+            <artifactId>postgresql</artifactId>
+            <version>42.4.0</version>
+        </dependency>
```
Comment (uladzimir-shelhunou-v2) [inline resource/pom.xml:82]: Move versions to a separate section (properties).

File: processor/src/test/resources/application.properties [hunk 3]
```diff
@@ -0,0 +1,8 @@
+spring.kafka.bootstrap-servers=${KAFKA_DOCKER_SERVICE:localhost}:9093
+resource-service-app.url=${RESOURCE_SERVICE_HOST:http://localhost:8091/}
+song-service-app.url=${SONG_SERVICE_HOST:http://localhost:8093/}
+
+kafka.consumer.group-id=songsapp
+kafka.topic-upload=resource-upload
+retry.backoff-period=2000
```
Comment (uladzimir-shelhunou-v2) [inline processor/src/test/resources/application.properties:7]: Always use exponential backoff.

https://dzone.com/articles/understanding-retry-pattern-with-exponential-back

**Review by uladzimir-shelhunou-v2 — CHANGES_REQUESTED** [review]

File: resource/pom.xml (hunk 2 above)
Comment (zarutski) [reply resource/pom.xml:87]: Done

File: processor/src/test/resources/application.properties (hunk 3 above)
Comment (zarutski) [reply processor/src/test/resources/application.properties:7]: Thanks, replaced

File: resource/src/main/resources/application.properties (hunk 1 above)
Comment (zarutski) [reply resource/src/main/resources/application.properties:1]: Will change it with the next task's implementation

**Pull request merged**